server/index.js

```diff
@@ -24,8 +26,21 @@ app.use('/auth', authRouter);
 app.use('/todoList', todoListRouter);
 app.use('/calendar', calendarRouter);
 
-app.get('', (req, res) => {
-  res.send('GET request');
+app.get('/info', (req, res) => {
+  const email = '1@qq.com';
```

Comment:
Cool. Is there going to be param or something to send back the "ownerEmail"?

Reply:
This email is use to locate the users table. The front-end should send the user's email through body or param, then the back end can get the email by req.body or req.param.

Reply:
Now i just set it to a constant value for testing